MyLibrary/Sources/AppFeature/AppView.swift

```diff
@@ -13,6 +13,10 @@ public struct AppReducer {
   @ObservableState
   public struct State: Equatable {
     var schedule = Schedule.State()
+    var liveTranslation = LiveTranslation.State(
+      roomNumber: ProcessInfo.processInfo.environment["LIVE_TRANSLATION_KEY"]
+        ?? (Bundle.main.infoDictionary?["Live translation room number"] as? String) ?? ""
+    )
```

Comment:
Extract to BulidConfig, which defined as Dependency Client.
See how it works at isowords:

https://github.com/pointfreeco/isowords/tree/main/Sources/Build

Build is preserved in .gitignore so use BuildConfig instead.